lib/ansible/modules/windows/win_feature.py

```diff
@@ -82,25 +82,27 @@
 '''
 
 EXAMPLES = r'''
-# This installs IIS.
-# The names of features available for install can be run by running the following Powershell Command:
-# PS C:\Users\Administrator> Import-Module ServerManager; Get-WindowsFeature
-$ ansible -i hosts -m win_feature -a "name=Web-Server" all
-$ ansible -i hosts -m win_feature -a "name=Web-Server,Web-Common-Http" all
-ansible -m "win_feature" -a "name=NET-Framework-Core source=C:/Temp/iso/sources/sxs" windows
```

Comment:
I see it for some, but sincerly I don't really see it for this one. Maybe @nitzmahone can say his opinion on this usage of this module.

Also, we should fix the contributing doc (I'll try to do it today/tomorrow)